src/scale.ts

```diff
@@ -660,6 +664,21 @@ export function channelScalePropertyIncompatability(channel: Channel, propName:
   throw new Error(`Invalid scale property "${propName}".`);
 }
 
+export function fieldDefMatchScaleType(specifiedType: ScaleType, fieldDefType: Type, bin?: boolean|BinParams):boolean {
```

Comment:
rename to `scaleTypeSupportDataType`  (there is no "fieldDef" anymore) and make `bin` required